test/extended/networking/isolation.go

```diff
@@ -7,13 +7,17 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-// This test requires a network plugin that supports namespace isolation.
-// NOTE: if you change the test description, update networking.sh too!
 var _ = Describe("[networking] network isolation plugin", func() {
 	f1 := e2e.NewFramework("net-isolation1")
 	f2 := e2e.NewFramework("net-isolation2")
 
-	It("should prevent communication between pods in different namespaces", func() {
+	var expectation string
+	if isMultitenant() {
```

Comment:
Trying to do this results in:

    /data/src/github.com/openshift/origin/test/extended/networking/isolation.go:58
    
    You may only call BeforeEach from within a Describe or Context

(I'm not calling BeforeEach at all.) It doesn't seem to like moving By() calls into a separate function from the It()/Describe().